.devcontainer/docker-compose.yml

```
@@ -1,8 +1,7 @@
-version: "3.8"
 services:
   cytex-texlive:
-    image: ran350/cytex-texlive:0.0.1
-    # build: . # if you want to build the image locally
+    # image: ran350/cytex-texlive:0.0.1
+    build: . # if you want to build the image locally
```

latestにしてなんかあったときにimageの更新だけで済むようにするのはどうでしょうか
(第三者の管理しているimageではないのでlatestでも良いかなと)

```suggestion
    image: tanuking551/cytex-texlive:0.0.1
    # build: . # if you want to build the image locally
```

---

Ubuntu version更新タスクの存在すっかり忘れてたわ、suggestありがとう！

前提確認したいんやけど、
- tanuking551はlionくんのアカウント？
- tanuking551/cytex-texlive:latestは、このPRからbuildしたimage？このリポジトリのDockerfileとcytex-texlive imageは合わせておきたいので確認

version latestにしてなかったのは、このリポジトリをforkして使ってる第三者のため、という経緯があるんすよね。メンテがすごく面倒になるとかがなければversion明記しておきたいなぁと思ってます。

---

> tanuking551はlionくんのアカウント？

です！

> tanuking551/cytex-texlive:latestは、このPRからbuildしたimage？このリポジトリのDockerfileとcytex-texlive imageは合わせておきたいので確認

です！ このPRのDockerfileから作りました

> version latestにしてなかったのは、このリポジトリをforkして使ってる第三者のため、という経緯があるんすよね。メンテがすごく面倒になるとかがなければversion明記しておきたいなぁと思ってます。

らじゃです。特にこだわりないので0.0.1を付けで対応しまんす

---

suggestion変更しておきました